app/src/components/LoadingScreen.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div class="text-center my-16">
+    <img style="width: 64px; height: 64px" class="mx-auto" src="/src/assets/spinner.svg" />
+    <h2 class="mt-6 text-3xl text-gray-500">
+      {{ text }}
+    </h2>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+  name: 'LoadingScreen',
```

Comment:
Nit: Rename to `LoadingSpinner` everywhere, since this is a component and not a full page screen

Reply:
Oooo nice one